scripts/configure.sh

```diff
@@ -41,63 +41,53 @@ if [[ "$OSTYPE" == "linux-gnu"* ]]; then
   apt update
   apt install -y build-essential wget cmake libgtest-dev lcov git software-properties-common rsync
 
-  # GitHub Actions in .github/workflows/validation.yml will attempt to cache and reuse leveldb built in this block.
-  # If a folder called leveldb-1.22 exists, skip the build step and go straight to install.
-  # See https://docs.github.com/en/free-pro-team@latest/actions/guides/caching-dependencies-to-speed-up-workflows
-  if [ ! -d "leveldb-1.22-${CMAKE_BUILD_TYPE}" ]; then
-    echo -e "${green}Building LevelDB from sources...${end}"
-    wget https://github.com/google/leveldb/archive/1.22.tar.gz
-    tar xzvf 1.22.tar.gz
-    rm -rf 1.22.tar.gz
-    mv leveldb-1.22 "leveldb-1.22-${CMAKE_BUILD_TYPE}"
-    cd "leveldb-1.22-${CMAKE_BUILD_TYPE}"
-    eval "cmake -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE} -DLEVELDB_BUILD_TESTS=0 -DLEVELDB_BUILD_BENCHMARKS=0 -DBUILD_SHARED_LIBS=0 ."
-    make -j$CPUS
-  else
-    echo -e "${green}Installing LevelDB from cache...${end}"
-    cd "leveldb-1.22-${CMAKE_BUILD_TYPE}"
-  fi
-  make install
-  cd ..
-
-  wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add -
-  add-apt-repository "deb http://apt.llvm.org/focal/ llvm-toolchain-focal-13 main"
-  apt install -y clang-format-13 clang-tidy-13
-  ln -s -f $(which clang-format-13) /usr/local/bin/clang-format
-  ln -s -f $(which clang-tidy-13) /usr/local/bin/clang-tidy
+  wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | $SUDO apt-key add -
+  $SUDO add-apt-repository "deb http://apt.llvm.org/focal/ llvm-toolchain-focal-14 main"
+  $SUDO apt install -y clang-format-14 clang-tidy-14
+  $SUDO ln -s -f $(which clang-format-14) /usr/local/bin/clang-format
+  $SUDO ln -s -f $(which clang-tidy-14) /usr/local/bin/clang-tidy
 fi
 
+LEVELDB_VERSION="1.23"
+echo -e "${green}Building LevelDB from sources...${end}"
+wget https://github.com/google/leveldb/archive/${LEVELDB_VERSION}.tar.gz
+rm -rf "leveldb-${LEVELDB_VERSION}-${CMAKE_BUILD_TYPE}"
+tar xzvf ${LEVELDB_VERSION}.tar.gz
+rm -rf ${LEVELDB_VERSION}.tar.gz
+mv leveldb-${LEVELDB_VERSION} "leveldb-${LEVELDB_VERSION}-${CMAKE_BUILD_TYPE}"
+cd "leveldb-${LEVELDB_VERSION}-${CMAKE_BUILD_TYPE}"
+eval "cmake -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE} -DLEVELDB_BUILD_TESTS=0 -DLEVELDB_BUILD_BENCHMARKS=0 -DBUILD_SHARED_LIBS=0 ."
```

Comment:
We had this line before and I suppose it just escaped me previously; why are we using `eval`? As far as I can tell, it shouldn't be needed at all, right?